README.md

```diff
@@ -15,9 +15,9 @@ worldengine world -s 1 -n seed1
 
 Worlds are generated using plate simulations, erosion, rain shadows, Holdridge life zones model and plenty of other phenomenons.
 
-Once a world it can be used for simulation civs evolution (see project [civs](https://github.com/ftomassetti/civs)).
+A finished world can be used for simulating the evolution of a civilization (see project [civs](https://github.com/ftomassetti/civs)).
```

Comment:
Or, you could just leave the word "finished" out, if you want to, and it
will still sound a lot better than the original.
On Nov 17, 2015 14:53, "Federico Tomassetti" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Mindwerks/worldengine/pull/202#discussion_r45135075>:
>
> > @@ -15,9 +15,9 @@ worldengine world -s 1 -n seed1
> >
> >  Worlds are generated using plate simulations, erosion, rain shadows, Holdridge life zones model and plenty of other phenomenons.
> >
> > -Once a world it can be used for simulation civs evolution (see project [civs](https://github.com/ftomassetti/civs)).
> > +A finished world can be used for simulating the evolution of a civilization (see project [civs](https://github.com/ftomassetti/civs)).
>
> Good then, I defer to the wisdom of native speaker :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mindwerks/worldengine/pull/202/files#r45135075>.
>